Build Cloud Account Id Artefacts 3.0.yml

```diff
@@ -0,0 +1,288 @@
+      inlineScript: >+
+        # Terminating the Instance
+
+        if('$(TerminateInstance)' -eq 'True') {
+           Write-Host "Removing the instance $(BuildImage.InstanceID) "
+            Remove-EC2Instance -InstanceId $(BuildImage.InstanceID) -Force
+        }
+
+        #Removing Vm
+
+        Write-Host "Removing the Vm"
+
+        Remove-EC2Instance -InstanceId  $(Vmtest.instanceID) -Force
+
+
+        # Deleting the Security Group
+
+        Write-Host "Deleting the security group"
+
+        Start-Sleep -Seconds 180
+         Remove-EC2SecurityGroup -GroupName 'w19d-15-0-$(VersionText-w19d-15-0)' -Force
+
+- job: Job_6
+  displayName: w19d-15-ID-NEW
+  timeoutInMinutes: 360
+  condition: and(succeeded(), or(eq(variables['Build-w16d-15-ID-NEW'], 'True'), eq(variables['Build-all'], 'True')))
+  pool:
+    vmImage: windows-2019
+  steps:
+  - checkout: self
+  - task: AmazonWebServices.aws-vsts-tools.AWSPowerShellModuleScript.AWSPowerShellModuleScript@1
+    displayName: Install PowerShell Modules
+    inputs:
+      awsCredentials: $(awsCredentials)
+      regionName: $(regionName)
+      filePath: scripts/install-powershell-modules.ps1
+      inlineScript: >+
+        if ("$(GitBranch)".Contains("refs/heads")) {
+            $branch ="$(GitBranch)".replace("refs/heads/", "")
+        } else {
+            $branch = "$(GitBranchPR)"
+        }
+
+        $branch | Write-Host | Out-Default | Write-Verbose
+
+        Write-Host "##vso[task.setvariable variable=GitBranch]$branch" | Out-Default | Write-Verbose
+
+        if ($Env:PsModuleInstalled -eq 'True') {
+            Write-Host "PS Module already installed, skip the Install PS Module Task." | Out-Default | Write-Verbose
+            return;
+        }
+
+        # Pester for Testing
+
+        Write-Host "Installing Module Pester RequiredVersion 5.6.1" | Out-Default | Write-Verbose
+
+        Install-Module -Name Pester -RequiredVersion 5.6.1 -AllowClobber -Force
+
+        Write-Host "Installed Module Pester" | Out-Default | Write-Verbose
+
+
+        $Env:PsModuleInstalled = 'True'
+
+  - task: DownloadSecureFile@1
+    name: keypair
+    displayName: Download KeyPair
+    inputs:
+      secureFile: AzureDevOps.pem
+  - task: AmazonWebServices.aws-vsts-tools.AWSPowerShellModuleScript.AWSPowerShellModuleScript@1
+    name: BuildImage
+    displayName: Build Image
+    inputs:
+      awsCredentials: $(awsCredentials)
+      regionName: $(regionName)
+      #arguments: " -VersionText \"$(System.JobDisplayName)\" -VersionMajor 15 -VersionMinor 0 -AmazonAMIName \"$(Windows-2019-AMI)\" -GitBranch \"$(GitBranch)\" -Cloud 'AWS' -Win2012 $false -KeyPairName $(KeypairName) -KeyPairPath $(keypair.secureFilePath) -GitUserName $(GitUserName) -RunWindowsUpdates $(RunWindowsUpdates) -ExternalIPAddresses  $(ExternalIPAddresses) -CloudAccountLicense 'x_lic_AWS*.5.lic' -InstallBaseSoftware $false -InstallScalable $false -Title 'LANSA Cloud Account Id New License'"

```

Comment:
Whats this here for?

Reply:
actually this build pipeline was not working so I copied from line from classic pipeline to compare. Anyway, I removed it.